src/test/java/unit/GcpNotebookControlled.java

```diff
@@ -190,6 +190,24 @@ void overrideLocationAndInstanceId() throws IOException {
         workspaceCreator.email.toLowerCase(),
         describeResource.privateUserName.toLowerCase(),
         "describe output matches private user name");
+
+    String newName = "NewOverrideLocationAndInstanceId";
+    String newDescription = "\"new override default location and instance id\"";
+    String newMetadata = "NewMetadata=metadata";
+    UFGcpNotebook updatedNotebook =
+        TestCommand.runAndParseCommandExpectSuccess(
+            UFGcpNotebook.class,
+            "resource",
+            "update",
+            "gcp-notebook",
+            "--name=" + name,
+            "--new-name=" + newName,
+            "--new-description=" + newDescription,
+            "--new-metadata=" + newMetadata);
```

Comment:
We need to document how this works when you only want to update some of the metadata, not all of it. Does this replace all the existing metadata, or append to it (if it's a new name)?

Reply:
Do you mean we will have many Metadata? 
I suppose we only have one metadata...

Reply:
it append